File: challenges/codeeval/easy/031/b1nary.cs
using System;
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
			string[] words = line.Split(',');
			int index = -1;
			for (int i=words[0].Length-1; i>=0; i--){
				if (words[1][0] == words[0][i]){
					index = i;
					break;
				}
			}
			Console.WriteLine(index);
		}
	}
}

File: challenges/codeeval/easy/031/b1nary.java
import java.io.*;
import java.util.*; 
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
			String[] words = line.split(",");
			int index = -1;
			for (int i=words[0].length()-1; i>=0; i--){
				if (words[1].charAt(0) == words[0].charAt(i)){
					index = i;
					break;
				}
			}
			System.out.println(index);
		}
	}
}